third_party/terraform/utils/config.go.erb

```diff
@@ -126,6 +126,10 @@ func (c *Config) loadAndValidate() error {
 
 	client := oauth2.NewClient(context.Background(), tokenSource)
 	client.Transport = logging.NewTransport("Google", client.Transport)
+    // Each individual request should return within 30s - timeouts will be retried.
+    // This is a timeout for, e.g. a single GET request of an operation - not a
+    // timeout for the maximum amount of time a logical request can take.
+    client.Timeout, _ = time.ParseDuration("30s")
 
```

Comment:
not positive if it matters but this looks like it has different indentation that the lines before it

Reply:
It doesn't matter in the output files (since it's compiled and `gofmt`'d), but it makes this file worse to look at - fixed.